Products/ZenRRD/zencommand.py

```diff
@@ -405,7 +414,9 @@ def doTask(self):
         self.state = SshPerformanceCollectionTask.STATE_CONNECTING
         try:
             if not self._manageIp:
-                raise Exception("Unknown host")                
+                self._eventService.sendEvent(self.manage_ip_event, severity=Event.Warning)
```

Comment:
In the non-ssh case, not having a manageIp is not necessarily any kind of problem.. The Openstack and OpenstackInfrastructure zenpacks are examples of this- they execute a command locally which connects to a URL specified in a zproperty, and have no mangageIp.

Reply:
you mean we should send events if this is True?
`if not self._manageIp and self._useSsh:`

Reply:
That would be fine for me.  I think otherwise you'd need to have a way to determine whether the manageIp is used.  (either manually specify it, or determine if it's passed to the command as an argument?)

Reply:
https://github.com/zenoss/zenoss-prodbin/pull/2812